testapp/src/main/res/xml/data_backup_rules.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<full-backup-content>
+  <exclude domain="sharedpref" path="."/>
```

Comment:
Excluding all shared preferences will also exclude the settings from the SDK side - like caching which mini apps have already been downloaded. So in that case, the `files` folder which includes the mini app assets would still be backed up to google cloud, but the shared preferences setting which caches that the mini app has been downloaded would not be backed up. So then the SDK would re-download the mini app, even though it's already been downloaded.

Could you specify just the file used by the sample app for the `path' here? i.e. `com.rakuten.tech.mobile.miniapp.sample.settings.xml`

Otherwise, we should just completely disable Google Backup for the App in order to avoid strange behavior like mentioned above.

Reply:
Yeah, I think `com.rakuten.tech.mobile.miniapp.sample.settings.xml` will work. However, I think, there is no good way to explore which data has been backed up, I was trying to find the answer in https://support.google.com/drive/answer/6305834.
Looks like the backup data will expire after a certain period, it depends on user when they like to backup their data. It means, from our side, it's challenging to rely on this backup data even if we want to use it in future. 

So, I am thinking if we should completely disable the Google backup for now.

Reply:
Okay, I think probably it's fine just to completely disable Google backup.

Reply:
Yeah, I have updated this PR.